packages/orientation-mixin/orientation-mixin.ts

```diff
@@ -0,0 +1,33 @@
+import { LitElement, property, PropertyValues } from 'lit-element';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T = object> = new (...args: any[]) => T;
+
+export interface OrientationInterface {
+  orientation: string;
+}
+
+export const OrientationMixin = <T extends Constructor<LitElement>>(
+  base: T
+): T & Constructor<OrientationInterface> => {
+  class Orientation extends base {
+    /**
+     * Set element disposition. Possible values are `horizontal|vertical`.
+     * @type {`horizontal|vertical}
+     */
+    @property({ type: String, reflect: true }) orientation = 'horizontal';
```

Comment:
We should avoid default values for properties, and use internal defaults when the public property isn't defined, right?

Reply:
Changed.